src/classes/HH_Container_LCTRL.cls

```diff
@@ -142,12 +170,24 @@ public class HH_Container_LCTRL {
                 //we use dynamic soql so we can include all contact fields, since custom naming may refer to any field.
                 list<string> listStrField = getContactFields();
                 string strSoql = 'SELECT ';
-                string strComma = '';
+                string strComma = '';                
+                set<String> setCheckUpdate = setUpdateableContactFields();
+                
                 for (string strF : listStrField) {
+
+                    // check field level security
+                    DescribeFieldResult dfr = UTIL_Describe.getFieldDescribe('Contact', strF);
```

Comment:
Is it possible for `strF` to be a relationship field?  For example, `Owner.Name` or something like that?  Would `UTIL_Describe.getFieldDescribe('Contact', 'Owner.Name')` fail?